admin/app/components/solidus_admin/stores/address_form/component.js

```diff
@@ -0,0 +1,56 @@
+import { Controller } from "@hotwired/stimulus"
```

Comment:
We already have a address form component that handles states. Can we use that one instead of implementing a second one?

Reply:
Yes, we can look into reusing the logic to pick a state. Does it make sense if everything else should be detached from the address table?